nautobot/dcim/forms.py

```diff
@@ -1712,31 +1712,33 @@ def __init__(self, *args, **kwargs):
 
                 # Collect interface IPs
                 # v2 TODO(jathan): Replace prefetch_related with select_related
-                interface_ips = (
-                    IPAddress.objects.ip_family(family)
-                    .filter(
-                        assigned_object_type=ContentType.objects.get_for_model(Interface),
-                        assigned_object_id__in=interface_ids,
-                    )
-                    .prefetch_related("assigned_object")
-                )
-                if interface_ips:
-                    ip_list = [(ip.id, f"{ip.address} ({ip.assigned_object})") for ip in interface_ips]
+                interface_ip_assignments = IPAddressToInterface.objects.filter(
+                    interface__in=interface_ids
+                ).prefetch_related("ip_address")
```

Comment:
This should be `select_related`, and then you can also ditch that `v2 TODO` comment. :) 

```suggestion
                ).select_related("ip_address")
```

Reply:
Covered by #3411